src/main/resources/public/css/main.css

```diff
@@ -58,16 +58,11 @@ a.btn-outline-primary:hover {
     height: 67px;
 }
 
-.gravatar img {
-    padding: 3px;
-    width: 100%;
-    border: 1px solid #ddd;
-    box-shadow: 1px 1px 3px #ccc;
-}
-
 .gravatar_header img {
     padding: 3px;
-    width: 100%;
+    width: 48px;
+    width: 48px;
```

Comment:
@criske You wrote ``width`` twice here. Did you want to say ``width`` and ``height``?

Reply:
@amihaiemil yes, is  `height` :D